### PR TITLE
Merging tips pages into Main

### PR DESCRIPTION
**Pull request by savvyp**

### CODE REVIEW

**Review by Arbanzo — COMMENTED** [review]
Formatting for tip page doesnt show proper, will edit

File: SafeFoodie_Fresh/lib/screens/authenticate/register.dart [hunk 1]
```diff
@@ -1,10 +1,11 @@
 import 'package:safefoodie_fresh/models/loginuser.dart';
 import 'package:safefoodie_fresh/services/auth.dart';
+import 'package:cloud_firestore/cloud_firestore.dart';
```
Comment (Arbanzo) [inline SafeFoodie_Fresh/lib/screens/authenticate/register.dart:3]: not needed, delete

File: SafeFoodie_Fresh/lib/screens/authenticate/register.dart [hunk 2]
```diff
@@ -13,7 +14,7 @@ class Register extends StatefulWidget {
 }
 
 class _Register extends State<Register> {
-  final AuthService _auth = new AuthService();
+  final AuthService _auth = AuthService();
```
Comment (Arbanzo) [inline SafeFoodie_Fresh/lib/screens/authenticate/register.dart:17]: delete, not used

**Review by Arbanzo — APPROVED** [review]
Formatting needs to be fixed but other than that the page works!

File: SafeFoodie_Fresh/lib/screens/home/MapSample.dart [hunk 3]
```diff
@@ -1,15 +1,16 @@
+// ignore_for_file: file_names
+
 import 'dart:async';
```
Comment (akmukhi) [inline SafeFoodie_Fresh/lib/screens/home/MapSample.dart:3]: I don't think we need this.

File: SafeFoodie_Fresh/lib/screens/home/home.dart [hunk 4]
```diff
@@ -1,65 +1,199 @@
...
+
   @override
   State<StatefulWidget> createState() {
     return _Home();
   }
 }
 
 class _Home extends State<Home> {
-  final AuthService _auth = new AuthService();
+  final AuthService _auth = AuthService();
```
Comment (akmukhi) [inline SafeFoodie_Fresh/lib/screens/home/home.dart:16]: I don't think this is used, delete.

File: SafeFoodie_Fresh/lib/screens/authenticate/register.dart (hunk 1 above)
Comment (savvyp) [reply SafeFoodie_Fresh/lib/screens/authenticate/register.dart:3]: done and done

File: SafeFoodie_Fresh/lib/screens/authenticate/register.dart (hunk 2 above)
Comment (savvyp) [reply SafeFoodie_Fresh/lib/screens/authenticate/register.dart:17]: done!

File: SafeFoodie_Fresh/lib/screens/home/MapSample.dart (hunk 3 above)
Comment (savvyp) [reply SafeFoodie_Fresh/lib/screens/home/MapSample.dart:3]: deleted

File: SafeFoodie_Fresh/lib/screens/home/home.dart (hunk 4 above)
Comment (savvyp) [reply SafeFoodie_Fresh/lib/screens/home/home.dart:16]: done

**Pull request closed without merging**